clang/include/clang/Frontend/CommandLineSourceLoc.h

```diff
@@ -89,8 +92,13 @@ struct ParsedSourceRange {
         // probably belongs to the filename which menas the whole
         // string should be parsed.
         RangeSplit.first = Str;
-      } else
+      } else {
+        // Column and line numbers are 1-based
```

Comment:
Ensure that full sentences in commends end with `.`

Remove brace for this single-line-simple-body case per https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements